projectPage.html

```diff
@@ -6,32 +6,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="HandheldFriendly" content="true">
-
     <title>PROJECTS</title>
 </head>
 
-<div class="all">
+<!-- <div class="all"> -->
 
-<body style="background: black;">
+<body class="all" style="background: black;">
 
 <div class="vC"> 
-<video class="vid" playsinline autoplay muted loop>
-    <source src="atlB&W.mp4" type="video/mp4">
-    </video>
-
-    <div>
-        <h1 class="pro">PROJECTS</h1>
-       </div>
-    </div>
-
-<!-- <style>
-    @import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap');
-    </style> -->
-
-<style>
-    @import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Oswald:wght@500&display=swap');
-    </style>
 
+<header>
```

Comment:
consolidating the video and the header in the same container means responsiveness based on container and relative units is easier to code/less code. its also good html hierarchy and enables use of the semantic element <header> so I really recommend making this change.